src/Services/ContentService.php

```diff
@@ -329,6 +333,25 @@ private function removePreviousSeenRecommendations($recommendations, $userId)
         return $filteredContent;
     }
 
+    private function removeNeedsAccessRecommendations($recommendations)
```

Comment:
Just to get more in the habit for typechecking content. We won't add docstrings for this.
```suggestion
    private function removeNeedsAccessRecommendations(array $recommendations) : array
```